watch.d/SOOP_ASF_MT.json

```diff
@@ -4,5 +4,5 @@
       "SOOP/ASF/MT"
   ],
   "execute": "bin/generic_incoming_handler.sh",
-  "execute_params": "--exec SOOP/SOOP_ASF_SST_XBT/soop_asf_sst_pipeline/dest_path.py --regex '^IMOS_SOOP-ASF_[A-Z]*_[0-9]{8}T[0-9]{6}Z_.*_FV0[0-1]_C-[0-9]{8}T[0-9]{6}Z.nc$' --email laurent.besnard@utas.edu.au"
+  "execute_params": "--exec SOOP/SOOP_ASF_SST_XBT/soop_asf_sst_pipeline/dest_path.py --regex '^IMOS_SOOP-ASF_[A-Z]*_[0-9]{8}T[0-9]{6}Z_.*_FV0[0-1].*_C-[0-9]{8}T[0-9]{6}Z.nc$' --email laurent.besnard@utas.edu.au"
```

Comment:
The regex could be even a bit more strict matching the file version field `_FV0[0-1]_.*`